src/main/resources/reference.conf

```diff
@@ -187,13 +190,13 @@ resourceTypes = {
     ownerRoleName = "owner"
     roles = {
       owner = {
-        roleActions = ["delete", "add", "view", "abort", "update", "alter_policies", "read_policies"]
+        roleActions = ["delete", "add", "view", "abort", "get_cost", "update", "alter_policies", "read_policies"]
       }
       reader = {
         roleActions = ["view"]
```

Comment:
should reader should have get_cost too?

Reply:
I'm not sure yet - I was thinking no and then we add it if people want it. My logic was - if someone doesn't own the workspace or have the ability to run anything in it - why do they care about the costs?

Reply:
is it complicated to remove an action from a role? If not, I don't mind readers having get_cost